doc/content/modules/user-manual/pages/release-notes/2025.12.0.adoc

```diff
@@ -24,6 +24,19 @@ Those parameters can be linked to other parameters using the _New Binding Connec
 
 image::sdv-action-parameters.png[Action parameters, width=65%,height=65%]
 
+- During textual export, the `OperatorExpressions` (mainly used in semantic `MetadataDefinition`) are now properly exported such as in:
```

Comment:
It's much more an enhancement than a new feature, so could you please move it in the appropriate section